example/code-snippets/i18n.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import DateRangePicker from 'react-daterange-picker';
+import moment from 'moment';
+
+/**
+ * Load moment locales that you need in your app
+ * to make the localeData available to the DatePicker
+ */
+require('moment/locale/en');
```

Comment:
i.e. `react-daterange-picker` by default loads in `require('moment/locale/en')`, so by default it is locale aware - and if you wish to change it in your application it's easier to do so